### DISCUSSION TITLE
Collapse long repeating messages

### DISCUSSION THREAD

**Discussion by carpinzer** [Chatterino/chatterino2#3177] [Ideas] [answered]
It would be useful to have a way to collapse long repeating messages to just one iteration of the message, for instance:

![xeTHMpgQsM](https://user-images.githubusercontent.com/89007739/129529473-a28308e6-3e04-4320-9108-f2bc865cd59f.png)

To just:

![xeTHMpgQsM](https://user-images.githubusercontent.com/89007739/129530664-a363bf6d-1a13-4053-bac0-b845d751e8ea.png)

Edit: The idea is to have this be independent of local r9k

**Comment by apa420** [comment, marked as answer]
You can use the Limit message height option for now.

**Reply by pajlada** [reply]
This could potentially be achieved with plugins too - see https://github.com/Chatterino/chatterino2/discussions/4999 for a discussion on plugin functionality.
